### PR TITLE
[FEATURE] Support emphasize-lines in code-block directive

### PR DESCRIPTION
**Pull request by linawolf**
resolves https://github.com/phpDocumentor/guides/issues/688

### CODE REVIEW

File: packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php [hunk 1]
```diff
@@ -101,4 +102,15 @@ private function setCaptionBasedOnOptions(array $options, CodeNode $node): void
 
         $node->setCaption($caption);
     }
+
+    /** @param DirectiveOption[] $options */
+    private function setEmphasizeLinesBasedOnOptions(array $options, CodeNode $node): void
+    {
+        $emphasizeLines = null;
+        if (isset($options['emphasize-lines'])) {
+            $emphasizeLines = (string) $options['emphasize-lines']->getValue();
```
Comment (jaapio) [inline packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php:111]: maybe we should validate the input?

**Review by jaapio — COMMENTED** [review]
Just a small suggestion, rather then that this looks fine

File: packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php (hunk 1 above)
Comment (linawolf) [reply packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php:111]: how would you validate it? regular expression?

Comment (jaapio) [reply packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php:111]: yes, that would do the trick, then log a warning when it is invalid.

Comment (linawolf) [reply packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php:118]: added a regex

**Review by jaapio — APPROVED** [review]

**Pull request merged**